twilight-model/src/guild/mod.rs

```diff
@@ -58,7 +58,6 @@ pub use self::{
     verification_level::VerificationLevel, widget::GuildWidget,
```

Comment:
Let's make the `member` module private and inline the `pub use self::member::Member` statement into the common pub use statement (and remove the outdated comment)